apsuite/orbcorr/si_bumps.py

```diff
@@ -36,7 +36,7 @@ class SiCalcBumps:
         'SB': _np.arange(2, 22, 2),
         'SP': _np.arange(3, 20, 4),
     }
-
+SECTION_TYPES = list(MARKER_NAMES.keys())
```

Comment:
@Gabrielrezende-asc, have you tested this? I think this identation is wrong, isn't it?

Reply:
besides, there is no empty line between this class constant and the definition of the method `__init__`.